src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -205,6 +205,15 @@ public static GitHub connectAnonymously() throws IOException {
     }
 
     /**
+     * Connects to GitHub Enterprise anonymously.
```

Comment:
Well, in this case we use the endpoint of a GH Enterprise server. I don't know where is the problem to add a new method.